docs/data/ampli/middleware.md

```diff
@@ -4,6 +4,9 @@ description: Use middleware to extend Amplitude by running a sequence of custom
 
 ---
 
+!!!note
+    Middleware is only supported in Ampli legacy.
```

Comment:
It might be good to mention/link to Plugins. e.g. Middleware is only supported in Ampli legacy. If you are on the latest Ampli this functionality has been replaced by [Plugins](url).